ipaserver/install/ipa_kra_install.py

```diff
@@ -83,33 +84,14 @@ def validate_options(self, needs_root=True):
     @classmethod
     def get_command_class(cls, options, args):
         if options.uninstall:
-            return KRAUninstaller
+            sys.exit(
+                'ERROR: Standalone KRA uninstallation was removed in '
+                'FreeIPA 4.5 as it had never worked properly and only caused '
```

Comment:
I would change the wording to `ERROR: Standalone KRA uninstallation was removed in FreeIPA 4.5` without extra information about how broken it was